src/main/java/frc/robot/commands/CmdFindAndGotoNoteCommand.java

```diff
@@ -0,0 +1,151 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.subsystems.VisionModule;
+import frc.robot.subsystems.noteplayer.NotePlayerSubsystem;
+import frc.robot.subsystems.swervedrive.SwerveDrive;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+
+public class CmdFindAndGotoNoteCommand extends Command {
+    private final NotePlayerSubsystem notePlayerSubsystem;
+    private final SwerveDrive swerveDrive;
+
+    private boolean finished = false;
+    private CommandXboxController driverController;
+    private PhotonCamera notesCamera;
+
+    public CmdFindAndGotoNoteCommand(CommandXboxController driverController, NotePlayerSubsystem notePlayerSubsystem, SwerveDrive swerveDrive) {
+        this.notePlayerSubsystem = notePlayerSubsystem;
+        this.swerveDrive = swerveDrive;
+        this.driverController = driverController;
+        // each subsystem used by the command must be passed into the
+        // addRequirements() method (which takes a vararg of Subsystem)
+        addRequirements(this.notePlayerSubsystem, this.swerveDrive);
+
+        notesCamera = VisionModule.getInstance().getNotesIndexerCamera();
+    }
+
+    Command defDriveCommand;
+
+    @Override
+    public void initialize() {
+        defDriveCommand = swerveDrive.getDefaultCommand();
+        swerveDrive.removeDefaultCommand();
```

Comment:
This is unnecessary, this command requires the swerve subsystem and will therefore automatically override the default command.

Reply:
Yes, I understand it is redundant.  It initially went in as an "hail mary" when having so many failed motor updates and loop overruns (while the roborio was in the process of dying).